src/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -1684,4 +1684,22 @@
   <data name="CryptoError_Sha1" xml:space="preserve">
     <value>Unexpected cryptographic error while computing SHA1 ({0}).</value>
   </data>
+  <data name="ClassLoad_General" xml:space="preserve">
+    <value>Could not load type '{0}' from assembly '{1}'.</value>
+  </data>
+  <data name="ClassLoad_MissingMethodRva" xml:space="preserve">
+    <value>Could not load type '{0}' from assembly '{1}' because the method '{2}' has no implementation (no RVA).</value>
```

Comment:
> some sort of `ValidateType` step 

I agree with the ValidateType-kind of thing, but I still believe that the bad RVA case should be handled as per-method failure and there is no need to differentiate it from other bad IL cases.

It is just a historic relict that the CLR type loader is verifying RVA .... it was actually worse originally: IIRC, there was even some verification of the IL blob done during type loading originally. I have removed it in 2007 because of it was just wasting time and memory.